bin/program.py

```diff
@@ -475,7 +476,7 @@ def run(self, bar, cwd, name, args=[]):
 
         if result.ok == -9:
             # Timeout -> stop retrying and fail.
-            bar.error(f'TIMEOUT after {timeout}s')
+            bar.log(f'TIMEOUT after {timeout}s',color=Fore.RED)
```

Comment:
why should this not be an error?

Reply:
I am actually not sure if this is the right way to handle this... However, I noticed that for example the visualizer does no error handling in this part of the code. Instead, the errors get handled in `VisualizerInvocation` in `generate.py`. The Generator handled it in both parts which lead to calling `bar.error` twice in case of a timeout which seemed to be wrong (i got a weird stack trace from the bar class)